docs/guides/scaling_crawlers.mdx

```diff
@@ -0,0 +1,126 @@
+
+import CodeBlock from '@theme/CodeBlock';
+
+import MinMaxConcurrencySource from '!!raw-loader!./scaling_crawlers_minMaxConcurrency.ts';
+import MaxRequestsPerMinuteSource from '!!raw-loader!./scaling_crawlers_maxRequestsPerMinute.ts';
+import AutoscaledPoolOptionsSource from '!!raw-loader!./scaling_crawlers_autoscaledPoolOptions.ts';
+
+As we build our crawler, we might want to control how many requests we do to the website at a time. Crawlee provides several options to fine tune how many parallel requests should be made at any time, how many requests should be done per minute, and how should scaling up and down happen based on the system resources available.
+
+:::tip
+
+All of these options are available on all crawlers Crawlee provides, but for this guide we'll be using the <ApiLink to="cheerio-crawler/class/CheerioCrawler">`CheerioCrawler`</ApiLink>. We can see all options that are available <ApiLink to="cheerio-crawler/interface/CheerioCrawlerOptions">`here`</ApiLink>
+:::
+
+## `minConcurrency` and `maxConcurrency`
+
+These control how many parallel requests can be run at any time. By default, crawlers will start with one parallel request at a time and scale up over time to a maximum of 1000 requests at a time.
+
+:::caution Don't set `minConcurrency` too high!
+
+Setting this option too high compared to the available system resources will make our crawler run extremely slow or might even crash.
+
+It's recommended to leave it at the default value that is provided and letting the crawler scale up and down automatically based on available resources instead.
+
+:::
+
+<CodeBlock language="js">
+	{MinMaxConcurrencySource}
+</CodeBlock>
+
+## `maxRequestsPerMinute`
+
+This controls how many total requests can be made per minute. It counts the amount of requests done every second, to ensure there is not a burst of requests at the `maxConcurrency` limit followed by a long period of waiting. By default, it is set to `Infinity` which means the crawler will keep going up to the `maxConcurrency`. We would set this if we wanted our crawler to work at full throughput, but also not keep hitting the website we're crawling with non-stop requests.
+
+<CodeBlock language="js">
+	{MaxRequestsPerMinuteSource}
+</CodeBlock>
+
+## Advanced configurations
+
+While the options above should be enough for most users, if we wanted to get super deep into the configuration of the autoscaling pool (the internal utility in Crawlee that helps us allow crawlers to scale up and down), we can do so through the <ApiLink to="cheerio-crawler/interface/CheerioCrawlerOptions#autoscaledPoolOptions">`autoscaledPoolOptions`</ApiLink> object available on crawler options.
+
+:::danger Complex options up ahead!
+
+This section is super advanced and, unless we test the changes extensively and know what we're doing, it's better to leave these options to the default configurations provided, as they are most likely going to work for us without much fuss.
```

Comment:
while generally i want to see the "we" in docs, here (or in general in hints like this) i'd say its better to keep using the "you". as the docs should be about telling a story, but hints are like "hey you, heads up!"

```suggestion
This section is super advanced and, unless you test the changes extensively and know what you're doing, it's better to leave these options to their defaults, as they are most likely going to work fine without much fuss.
```